lib/_http_client.js

```diff
@@ -430,13 +430,13 @@ function socketOnEnd() {
 }
 
 function socketOnData(d) {
-  var socket = this;
-  var req = this._httpMessage;
-  var parser = this.parser;
+  const socket = this;
+  let req = this._httpMessage;
+  let parser = this.parser;
 
   assert(parser && parser.socket === socket);
 
-  var ret = parser.execute(d);
+  let ret = parser.execute(d);
```

Comment:
I'm not sure I see where this is changing. `const`?

Reply:
thank you for your suggest!